### DISCUSSION TITLE
posts/tip-day-sharepoint-debug-console-ctrl-f12/

### DISCUSSION THREAD

**Discussion by giscus** [ValerasNarbutas/ValerasNarbutas.github.io#9] [Comments]
# posts/tip-day-sharepoint-debug-console-ctrl-f12/

TIP: Did you know you can access SharePoint debug console by using Ctrl+f12 ?

https://valerasnarbutas.github.io/posts/tip-day-sharepoint-debug-console-ctrl-f12/

**Comment by fonbrauzen** [comment]
The key point here "on modern pages" - [from here](https://learn.microsoft.com/en-us/sharepoint/dev/spfx/use-developer-dashboard)

**Reply by ValerasNarbutas** [reply]
yes, indeed! @fonbrauzen 
i will update post  ;)
